packages/jss-keyframes-helper/src/index.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import defaultJss, {SheetsRegistry, type Jss, type StyleSheet, type StyleRule} from 'jss'
+
+type Options = {
+  jss?: Jss,
+  name?: string,
+  sheet?: StyleSheet,
+  registry?: SheetsRegistry
+}
+
+export const sheetsRegistry = new SheetsRegistry()
```

Comment:
I would seriously consider to use always one styleSheet, generate and attach it lazily once createKeyframes is called the first time and export it

Reply:
I added the multiple stylesheets for different jss versions. They are lazily attached and reused between the same jss instance though.